src/MemoryTracing/include/MemoryTracing/MemoryInfoListener.h

```diff
@@ -5,6 +5,10 @@
 #ifndef MEMORY_TRACING_MEMORY_INFO_LISTENER_H_
 #define MEMORY_TRACING_MEMORY_INFO_LISTENER_H_
 
+#include <absl/base/internal/thread_annotations.h>
```

Comment:
I think this should be `#include <absl/base/thread_annotations.h>` (without the `internal`).

Reply:
Fixed! Thanks !